openmrs/apps/registration/app.json

```diff
@@ -30,19 +30,20 @@
         "patientHIVTestInformation": {
                 "title": "patientHIVTestInformation",
                 "translationKey": "REGISTRATION_HIVTEST_INFO_KEY",
-                "attributes": ["DateofHIVDiagnosis","Typeoftest","TestResult","Patient Status"],
+                "attributes": ["DateofHIVDiagnosis","Typeoftest","TEST_RESULT","PATIENT_STATUS","LOCATION_OF_TEST","SECTOR_SELECT","ATIP_SELECT","HEALTH_FACILITY_NAME","HEALTH_FACILITY_DISTRICT","HEALTH_FACILITY_PROVINCE"],
```

Comment:
@krishnanspace where can I find these person attributes to test with? I have looked on QA but they are not there.